### ISSUE TITLE
Incomplete parsing of plugins field in results

### ISSUE THREAD

**Issue by ShaktiKShrivastava** [AlienVault-OTX/OTX-Python-SDK#40]
Only few plugins are being parsed in the result that's being returned in 
`result = otx.get_indicator_details_full(hash_type, hash)`
 Fields like msdefender, exiftool and some others aren't being taken into consideration in the def file(otx, hash) function. This is leading to some hashes not being recognised as malicious even though they have been listed as malicious on OTX site

**Comment by rustybrooks** [comment]
Yeah this is probably a case of the state of the art having advanced on the server side, and the SDK not completely having kept up.  I'll look into it.
